kore/src/Kore/Log/KoreLogOptions.hs

```diff
@@ -199,9 +203,24 @@ readSeverity =
         "error"    -> pure Error
         _          -> Nothing
 
+parseWarningSwitch :: Parser WarningSwitch
+parseWarningSwitch =
+    Options.flag
+        AsWarning
+        AsError
+        ( Options.long "Werror"
```

Comment:
The frontend uses `--warnings-to-errors`, so that may be more familiar to people. Unfortunately, this will parse `--Werror`, not `-Werror`, I think.